content/20.treatments.md

```diff
@@ -286,7 +286,32 @@ When these identifiers are followed back to the registry, most trials claim they
+The secondary end points were amelioration of the symptoms of the disease such as axillary temperature ≤36.6°C,SpO2 >94% on room air,and disappearance of symptoms like shortnes of breath, cough, sore throat.
+The median time to symptom alleviation was similar across conditions (19 days in HCQ+SOC vs 21 days in SOC). 
+However, the investigators reported an interesting finding revealed in _post hoc_ analysis: controlling for the administration of anti-virals yielded an effect of HCQ on the alleviation of symptoms, with a reported hazard ratio of 8.83.
+However, the 95% confidence interval of 1.09 to 71.3 suggests that this analysis may be under-powered and should be interpreted cautiously, as only 28 patients total (14 in each group) met this criterion.
+Additionally, there was a non-significant trend towards a greater number of lymphocytes in the SOC+HCQ group compared to the SOC alone group (mean of 0.062 versus 0.008; p=0.57). 
+Given the improvement in CRP levels and the possible improvement in lymphocyte count, the authors hypothesized that the addition of HCQ to the current SOC could decrease the inflammatory response and subsequently prevent multiorgan failure and death. 
+Additionally, one of the key results of this trial was that the 30% of the patients receiving SOC+HCQ reported adverse outcomes as opposed to 8.8% of patients receiving only SOC. 
+The most common adverse outcome in the SOC+HCQ group was diarrhea (10% vs 0% in the SOC group, p=0.004). 
+
+The study had a few limitations. 
+One of the limitations of the study was that the cohort mostly consisted of patients with mild/moderate disease symptoms, and the average patient in the analysis was 46.
+This may not be representative of COVID-19 patients broadly because older age groups are known to be at higher risk.
+Although the authors claimed that HCQ could have beneficial effects in hindering disease progression, multiorgan failure, and death in COVID-19, this finding cannot be extrapolated to older patients or severe cases based on the evidence they present.
+Indeed, a larger sample needs to be tested to validate whether lymphocyte counts do increase in patients treated with SOC+HCQ; although the result in the present analysis appeared promising, it lacked statistical significance.
+Additionally, in this study, SOC included the use of antivirals (Lopinavir-Ritonavir, Arbidol, Oseltamivir, Virazole, Entecavir, Ganciclovir and/or Interferon-alpha), which appeared to introduce confounding effects. 
+Thus, to better assess the effects of HCQ, an alternate study design needs to be outlined where SOC does not involve the use of anti-virals.
+In this trial, the samples used to test for presence of the SARS-CoV-2 virus were collected from the upper respiratory tract, and the authors indicated that the use of upper respiratory samples may have introduced false negatives; thus, biomarkers for the disease are needed. 
```

Comment:
```suggestion
In this trial, the samples used to test for the presence of the SARS-CoV-2 virus were collected from the upper respiratory tract and the authors indicated that the use of upper respiratory samples might have introduced false negatives.
```
What testing method did they use in particular? If it was some kind of PCR testing, what's the explanation that PCR from upper respiratory collection is prone to false negativeness?

Reply:
They used RT-PCR. The authors state the use of upper respiratory samples as a limitation, which seems like it could be a valid concern (e.g., [this paper](https://jamanetwork.com/journals/jama/article-abstract/2762997)) because that study found that it is easiest to detect the virus in lower respiratory tract samples. They didn't have LR samples because the patients were not severe enough to warrant collection.

Perhaps we could add that citation?
```suggestion
In this trial, the samples used to test for presence of the SARS-CoV-2 virus were collected from the upper respiratory tract, and the authors indicated that the use of upper respiratory samples may have introduced false negatives (e.g., [@doi:10.1001/jama.2020.3786]); thus, the identification of biomarkers that can be collected non-invasively would be valuable to studies such as this one.
```

(I'm answering some of these questions because some of the concerns you raise are about suggestions I made on @soumitagh's text!)